apps/src/code-studio/pd/professional_learning/RegionalWorkshopCatalog.jsx

```diff
@@ -20,15 +20,21 @@ import RegionalWorkshopCatalogCard from './RegionalWorkshopCatalogCard';
 
 import style from './regionalWorkshopCatalog.module.scss';
 
-export default function RegionalWorkshopCatalog({zipFromSchoolInfo}) {
+export default function RegionalWorkshopCatalog({
```

Comment:
Many of the changes in this file are just adding containers and moving chunks around so I'd recommend hiding whitespace!